.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
```

Comment:
Definitely useful to have a code style set. Any particular reason for choosing 2 spaces instead of 4?

Reply:
That's what we use in Vert.x 😉 

I have personally been using 2 spaces for years, but it's really a matter of preferences.

There are other code style guides advocating 2 spaces, see https://google.github.io/styleguide/javaguide.html as an example.